bridge/engine/EngineMixer.cpp

```diff
@@ -1837,6 +1836,17 @@ void EngineMixer::processIncomingRtpPackets(const uint64_t timestamp)
     {
         ++numRtpPackets;
 
+        const auto rtpHeader = rtp::RtpHeader::fromPacket(*packetInfo.packet());
+        if (!rtpHeader)
+        {
+            continue;
+        }
+        auto inboundSsrcContext = getInboundSsrcContext(rtpHeader->ssrc.get());
```

Comment:
Just a suggestion. Maybe we should store `inboundSsrcContext` inside of `IncomingPacketInfo` instead of `transport`. Seems like the jobs that will enqueue the `IncomingPacketInfo` already have a reference to `inboundSsrcContext` (exception is maybe the RTCP packets but we can obtain the inboundSsrc before enqueue) and `inboundSsrcContext` already have a pointer to to `transport`. In this way we avoid `getInboundSsrcContext` twice (before and after enqueue) and avoid this pattern of getting and checking if it is not a `nullptr` in almost `process` methods